config/locales/server.en.yml

```
@@ -2344,6 +2344,7 @@ en:
     create_revision_on_bulk_topic_moves: "Create revision for first posts when topics are moved into a new category in bulk."
 
     allow_changing_staged_user_tracking: "Allow a staged user's category and tag notification preferences to be changed by an admin user."
+    use_email_for_username_and_name_suggestions: "Use email for username and name suggestions. Please note that enabling this setting is dangerous since using emails for username suggestions makes user emails easily guessable."
```

Not sure about the language here, also not sure if this warning is scary enough. Would appreciate any feedback.

---

I think the clearest way will be to include an example like `gmail.com`. Maybe something like:
```suggestion
    use_email_for_username_and_name_suggestions: "Use the first part of email addresses for username and name suggestions. Note that this makes it easier for the public to guess full user email addresses (because a large proportion of people share common services like `gmail.com`).
```

---

This is much better, thank you!